roles/routeros_common/tasks/manage_users.yml

```diff
@@ -0,0 +1,102 @@
+---
+# Manage Mikrotik Users
+# Missing users are added with random password and ssh public key loaded.
+# Users present and in the delete_users list are delete.
+# For now, we leave extra users as is.
+
+    - name: setup expected users
+      set_fact:
+        expected_users: "{{ users_admin + users_user + ['admin'] }}"
+        delete_users: ['DeleteMe1', 'DeleteMe2']
```

Comment:
this list looks like test code, so I'm surprised to see it outside the molecule tree.

Reply:
I have removed the hard-coded list and added a new user list for deletions (users_delete).